lib/config/options/index.ts

```diff
@@ -223,8 +223,7 @@ const options: RenovateOptions[] = [
     description: `Enable or disable Renovate bot.`,
     stage: 'package',
     type: 'boolean',
-    cli: false,
-    env: false,
+    cli: true,
```

Comment:
i think this can be removed too, as true is the default?

https://github.com/renovatebot/renovate/blob/e5336a552659ad3f29eeb484d550d8e90970e74e/lib/workers/global/config/parse/cli.ts#L11

```suggestion
```